### ISSUE TITLE
add 'id' to r.db.info() and r.table.info()

### ISSUE THREAD

**Issue by larkost** [rethinkdb/rethinkdb#3358]
It would be very helpful if the `.info()` results for both tables and databases would return the respective item's `id` (uuid) value. This would significantly shorten getting the value, e.g.:

```python
# current shortest methods
r.db('test').table_config('a').nth(0)['id'].run()
r.db('rethinkdb').table('db_config').filter({'name':'test'}).nth(0)['id'].run()

# with change
r.db('test').table('a').info()['id'].run()
r.db('test').info()['id'].run()
```

**Comment by Tryneus** [comment]
This is up in review 2460.  Added `id` to `table.info()` and `db.info()` - this field will be `null` for system db/tables.  I think the discussion over treating system and db tables identically is a separate issue.

**Comment by danielmewes** [comment]
Sounds good @Tryneus .

**Comment by Tryneus** [comment]
Fix merged to `next` in commit 163951e.  Will be in release 1.16.
